src/interface/src/styles/_mixins.scss

```diff
@@ -101,41 +97,19 @@ Declare mixins based on styleguide and apply to classes
   line-height: 32px;
 }
 
-@mixin button-text {
```

Comment:
are this mixins not in use?

Reply:
some of these were dupes in the file, but I'll doublecheck all of these one more time before I merge.

Reply:
FWIW, we already had small-paragraph (line 41) regular-paragraph (line 33) here, and I moved button-text down below, since that's where other form/button related mixins are. I'm surprised the linter doesn't catch dupes, but I suppose it needs to allow them.